packages/drawer/Drawer.tsx

```diff
@@ -22,28 +24,41 @@ export class Drawer extends Component<DrawerProps> {
   static get props() {
     return {
       position: prop.string( {
-        attribute: true
+        attribute: {
+          source: true
+        }
       } ),
       visible: prop.boolean( {
-        attribute: true
+        attribute: {
+          source: true
+        }
+      } ),
+      floating: prop.boolean( {
+        attribute: {
+          source: true
+        },
+        initial: true
       } )
     };
   }
 
   position = 'top';
```

Comment:
Also pls don't use `initial`, this is deprecated and will be removed in next version of skate. If you have any questions about skate API don't hesitate to ask me ( as I am core contributor and know internals and what not )